src/install/lockfile.zig

```diff
@@ -3215,10 +3329,6 @@ pub const Serializer = struct {
         try writer.writeIntLittle(u64, 0);
         const end = try stream.getPos();
 
-        try writer.writeIntLittle(u64, this.workspace_path.len);
```

Comment:
we cannot remove this code without bumping the lockfile version

this corrupts existing lockfiles

the idea with workspace_path was we'd store a relative path to the parent-most package.json for the workspace

so we'd know "is this a workspace?"

if it doesn't turn out to be useful, we can just waste the 8 bytes there and skip it

Reply:
Ah okay - I thought since this is the end of file anyway, dropping it will blend into the alignment padding hence no observable side effects.